README.md

```diff
@@ -69,18 +69,18 @@ Additionally you can pass a JSONArray as a String in order to send complex
 messages, as per the example:
 
 ```
-import net.sf.json.JSONArray;
-import net.sf.json.JSONObject;
-node {
-    JSONArray attachments = new JSONArray();
-    JSONObject attachment = new JSONObject();
-
-    attachment.put('text','I find your lack of faith disturbing!');
-    attachment.put('fallback','Hey, Vader seems to be mad at you.');
-    attachment.put('color','#ff0000');
+import groovy.json.JsonOutput
 
-    attachments.add(attachment);
-    slackSend(color: '#00FF00', channel: '@gustavo.maia', attachments: attachments.toString())
+node {
+    def attachments = [
+        [
+            text: 'I find your lack of faith disturbing!',
+            fallback: 'Hey, Vader seems to be mad at you.',
+            color: '#ff0000'
+        ]
+    ]
+
+    slackSend(color: '#00FF00', channel: '@gustavo.maia', attachments: JsonOutput.toJson(attachments))
```

Comment:
Does this require any whitelisting or it works OOTB?

Reply:
you're right, seems I had a whitelisting already configured for JsonOutput.toJson. Not optimal. Converstion to JSON should be handle in slackSend. I'll try to find time to propose a patch for it.

Reply:
please do, that would be great <3